### DISCUSSION TITLE
Add back SearX as a available search engine

### DISCUSSION THREAD

**Discussion by chrislogan-foss** [ungoogled-software/ungoogled-chromium-android#112] [Ideas]
I would like to know why Searx.me was removed as a option in the search engine menu and if it will ever come back.

**Comment by wchen342** [comment]
It was removed because SearX main instance no longer exists. See https://github.com/wchen342/ungoogled-chromium-android/issues/19 and https://github.com/Eloston/ungoogled-chromium/issues/1041.
